geniza/corpus/management/commands/sync_annotation_export.py

```diff
@@ -1,22 +1,132 @@
+import json
+import os.path
+from collections import defaultdict
+from datetime import datetime
+
 from django.conf import settings
+from django.contrib.admin.models import LogEntry
+from django.contrib.contenttypes.models import ContentType
 from django.core.management.base import BaseCommand
+from django.utils import timezone
 
+from geniza.annotations.models import Annotation
 from geniza.corpus.annotation_export import AnnotationExporter
+from geniza.corpus.models import Document
 
 
 class Command(BaseCommand):
     """Synchronize annotation backup data with GitHub"""
 
+    # filename for last run information (stored in current user's home)
+    lastrun_filename = os.path.join(os.path.expanduser("~"), ".pgp_export_lastrun")
```

Comment:
Does this need to live outside of the project directory tree? I would have put it somewhere like `data/` but maybe that won't work?

Reply:
Hmm. I'm adapting this from code I wrote for PEMM, so I just did something similar to that. It doesn't need to be, but I feel like home directory might be a bit more durable than data. I think we're always going to be running it as the deploy user (or ourselves for dev), so it doesn't need to be in a shared location across accounts.

Reply:
That makes sense!